### PR TITLE
Tasks Verify

### PR DESCRIPTION
**Pull request by amyrchyk**

### CODE REVIEW

File: src/main/java/com/qaprosoft/carina/demo/mobile/gui/pages/android/LoginPage.java [hunk 1]
```diff
@@ -79,4 +79,42 @@ public CarinaDescriptionPageBase login() {
...
+       	
+      	return privacyPolicyCheckbox.isChecked();
+      }
+      public boolean isSexCheck() {
+      	if(maleRadioBtn.isChecked()|femaleRadioBtn.isChecked()) {
+      		return true;
+      	}
+    		return false;
+       	
+      }
```
Comment (vrudyk) [inline src/main/java/com/qaprosoft/carina/demo/mobile/gui/pages/android/LoginPage.java:119]: a lot of unnecessary spaces
see above methods and remove unnecessary spaces

File: src/main/java/com/qaprosoft/carina/demo/mobile/gui/pages/android/LoginPage.java [hunk 2]
```diff
@@ -79,4 +79,42 @@ public CarinaDescriptionPageBase login() {
...
+      public boolean isPrivacyPolicyCheckboxCheck() {
+       	
+      	return privacyPolicyCheckbox.isChecked();
+      }
+      public boolean isSexCheck() {
+      	if(maleRadioBtn.isChecked()|femaleRadioBtn.isChecked()) {
+      		return true;
+      	}
+    		return false;
```
Comment (vrudyk) [inline src/main/java/com/qaprosoft/carina/demo/mobile/gui/pages/android/LoginPage.java:117]: better verify only one sex which you pick in test method

File: src/main/java/com/qaprosoft/carina/demo/mobile/gui/pages/common/LoginPageBase.java [hunk 3]
```diff
@@ -23,5 +23,22 @@ public LoginPageBase(WebDriver driver) {
...
+	public abstract boolean isFieldPassPresent();
+	
+	public abstract boolean isMaleSexRadioButtonPresent();
+	
+	public abstract boolean isFemaleSexRadioButtonPresent();
+	
+	public abstract boolean isPrivacyPolicyCheckboxPresent();
+	
+	public abstract boolean isMaleSexRadioButtonCheck();
+	public abstract boolean isFemaleSexRadioButtonCheck();
```
Comment (vrudyk) [inline src/main/java/com/qaprosoft/carina/demo/mobile/gui/pages/common/LoginPageBase.java:38]: add space

File: src/test/java/com/qaprosoft/carina/demo/MobileSampleTest.java [hunk 4]
```diff
@@ -85,4 +85,62 @@ public void testUIElements() {
...
+    public void testVerify() {
+        WelcomePageBase welcomePage = initPage(getDriver(), WelcomePageBase.class);
+        Assert.assertTrue(welcomePage.isPageOpened(), "Welcome page isn't opened");
+        LoginPageBase loginPage = welcomePage.clickNextBtn();
+        Assert.assertFalse(loginPage.isLoginBtnActive(), "Login button is active when it should be disabled");
+        Assert.assertTrue(loginPage.isFieldNamePresent(),"Field isn't presented");
+        Assert.assertTrue(loginPage.isFieldPassPresent(),"Field isn't presented");
+        Assert.assertTrue(loginPage.isMaleSexRadioButtonPresent(),"RadioButton Male isn't presented");
+        Assert.assertTrue(loginPage.isFemaleSexRadioButtonPresent(),"RadioButton Female isn't presented");
+        Assert.assertTrue(loginPage.isPrivacyPolicyCheckboxPresent(),"CheckBox Privacy isn't presented");
```
Comment (vrudyk) [inline src/test/java/com/qaprosoft/carina/demo/MobileSampleTest.java:100]: add space between , and "

File: src/test/java/com/qaprosoft/carina/demo/MobileSampleTest.java [hunk 5]
```diff
@@ -85,4 +85,62 @@ public void testUIElements() {
...
     }
 
+    @Test(description = "JIRA#DEMO-0001")
+    @MethodOwner(owner = "amyrchyk")
+    @TestLabel(name = "feature", value = {"mobile", "acceptance"})
+    public void testVerify() {
+        WelcomePageBase welcomePage = initPage(getDriver(), WelcomePageBase.class);
+        Assert.assertTrue(welcomePage.isPageOpened(), "Welcome page isn't opened");
+        LoginPageBase loginPage = welcomePage.clickNextBtn();
+        Assert.assertFalse(loginPage.isLoginBtnActive(), "Login button is active when it should be disabled");
```
Comment (vrudyk) [inline src/test/java/com/qaprosoft/carina/demo/MobileSampleTest.java:95]: this assert must be below
read task2 and move

File: src/test/java/com/qaprosoft/carina/demo/MobileSampleTest.java [hunk 6]
```diff
@@ -85,4 +85,62 @@ public void testUIElements() {
...
+        Assert.assertFalse(loginPage.isMaleSexRadioButtonCheck(), "Male is checked");
+        Assert.assertFalse(loginPage.isFemaleSexRadioButtonCheck(), "Female is checked");
+        Assert.assertFalse(loginPage.isPrivacyPolicyCheckboxCheck(), "Privacy is checked");
+        String username = "Test user";
+        String password = RandomStringUtils.randomAlphabetic(10);
+        loginPage.typeName(username);
+        loginPage.typePassword(password);
+        loginPage.selectMaleSex();
+        loginPage.checkPrivacyPolicyCheckbox();
+        Assert.assertTrue(loginPage.isSexCheck(), "Sex isn't checked");
```
Comment (vrudyk) [inline src/test/java/com/qaprosoft/carina/demo/MobileSampleTest.java:110]: use  isMaleSexRadioButtonCheck

File: src/test/java/com/qaprosoft/carina/demo/MobileSampleTest.java [hunk 7]
```diff
@@ -85,4 +85,62 @@ public void testUIElements() {
...
+        loginPage.typeName(username);
+        loginPage.typePassword(password);
+        loginPage.selectMaleSex();
+        Assert.assertTrue(loginPage.isSexCheck(), "Sex isn't checked");
+        Assert.assertTrue(loginPage.isPrivacyPolicyCheckboxCheck(), "Privacy isn't checked");
+        Assert.assertFalse(loginPage.isLoginBtnActive(), "Button Sign Up is active");
+
+        
+        
+    }
```
Comment (vrudyk) [inline src/test/java/com/qaprosoft/carina/demo/MobileSampleTest.java:144]: you can add one new assert in first method and remove this

File: src/main/java/com/qaprosoft/carina/demo/mobile/gui/pages/android/LoginPage.java (hunk 1 above)
Comment (amyrchyk) [reply src/main/java/com/qaprosoft/carina/demo/mobile/gui/pages/android/LoginPage.java:119]: done

File: src/main/java/com/qaprosoft/carina/demo/mobile/gui/pages/android/LoginPage.java (hunk 2 above)
Comment (amyrchyk) [reply src/main/java/com/qaprosoft/carina/demo/mobile/gui/pages/android/LoginPage.java:117]: done

File: src/main/java/com/qaprosoft/carina/demo/mobile/gui/pages/common/LoginPageBase.java (hunk 3 above)
Comment (amyrchyk) [reply src/main/java/com/qaprosoft/carina/demo/mobile/gui/pages/common/LoginPageBase.java:38]: done

File: src/test/java/com/qaprosoft/carina/demo/MobileSampleTest.java (hunk 4 above)
Comment (amyrchyk) [reply src/test/java/com/qaprosoft/carina/demo/MobileSampleTest.java:100]: done

File: src/test/java/com/qaprosoft/carina/demo/MobileSampleTest.java (hunk 5 above)
Comment (amyrchyk) [reply src/test/java/com/qaprosoft/carina/demo/MobileSampleTest.java:95]: done

File: src/test/java/com/qaprosoft/carina/demo/MobileSampleTest.java (hunk 6 above)
Comment (amyrchyk) [reply src/test/java/com/qaprosoft/carina/demo/MobileSampleTest.java:110]: done

File: src/test/java/com/qaprosoft/carina/demo/MobileSampleTest.java (hunk 7 above)
Comment (amyrchyk) [reply src/test/java/com/qaprosoft/carina/demo/MobileSampleTest.java:144]: done

**Comment by vrudyk** [comment]
ok

**Pull request merged**